### DISCUSSION TITLE
Rebooting marbits master node NOW

### DISCUSSION THREAD

**Discussion by pablo-icm** [marbits-icm/marbits-public#4] [Announcements]
Dear users,

marbits master node load is skyrocketing due to lots of input output queued processes (`/mnt/lustre` is /mnt/lustreing again). See issue #2. I need to reboot it urgently, so I have had to restrict your access while I manage it. I hope it's back online soon, and when it is, lustre will be unmounted in hopes of avoiding further problems.

Cheers

**Comment by pablo-icm** [comment]
Marbits master node has been rebooted, but with no access to `/mnt/lustre` for the time being. Sorry about that. Your jobs should be running with no issues, but as always double check your log files and results.

I leave this conversation open so as if anything arises (you can't log in, you still experience strange behaviours, etc...) you can report it here.

**Reply by lidmontiel** [reply]
Thanks a lot, Pablo.

It's working now! 

P.S: my `.bashrc` seems to have been edited today. Did someone do it? 👀

**Reply by pablo-icm** [reply]
🫣 I might have tried a couple of things before rebooting... I'm sure you have already uncommented those lines
